assets/css/style.css

```diff
@@ -400,7 +400,6 @@ input {
   .article-item {
```

Comment:
- [ ] Good job with the design 🥇 I think it would b better to add a bit more margin between `headline` section and `project` section to get a design close to the Figma template 🤓⛏ 

### **_Desktop version_** 

Your design | Figma template
---| ---|
![image](https://github.com/WilfriedM01/portfolio/assets/106119105/595bb2f8-9c46-447c-9b6c-37a9682209bd) | ![image](https://github.com/WilfriedM01/portfolio/assets/106119105/7f0ac8e0-d7c7-4e16-8655-4a494deee1d6)


### **_Mobile version_** 
Your design | Figma template
---| ---|
![image](https://github.com/WilfriedM01/portfolio/assets/106119105/a7cc7d0f-f84b-40ec-a6d5-ce10c937de17) | ![image](https://github.com/WilfriedM01/portfolio/assets/106119105/992ff823-d73e-431c-a2c8-c0ad25756b9f)